test/test.go

```diff
@@ -505,6 +505,10 @@ func (h *Helper) Path(name string) string {
 		joined = filepath.Join(h.tempdir, name)
 	}
 
+	if _, err := os.Stat(joined); os.IsNotExist(err) {
+		return ""
+	}
```

Comment:
If this is to help with nonexistent `final` files, I think I'd move this to CompareFile.  I'm ok changing the logic there to be "don't check file if not in `final`".